docs/usage-examples/insertMany.txt

```diff
@@ -24,40 +24,78 @@ To insert multiple documents, call the ``insert()`` method and specify the new d
 as an array inside the method call. Each array entry contains a single document's field
 values.
 
-Example
--------
-
-This usage example performs the following actions:
-
-- Uses the ``Movie`` Eloquent model to represent the ``movies`` collection in the
-  ``sample_mflix`` database
-- Inserts documents into the ``movies`` collection
-- Prints whether the insert operation succeeds
-
-The example calls the ``insert()`` method to insert documents that contain
-information about movies released in ``2023``. If the insert operation is
-successful, it returns a value of ``1``. If the operation fails, it throws
-an exception.
-
-.. io-code-block::
-   :copyable: true
+.. tip::
 
-   .. input:: ../includes/usage-examples/InsertManyTest.php
-      :start-after: begin-insert-many
-      :end-before: end-insert-many
-      :language: php
-      :dedent:
+   To learn more about insert operations, see the
+   :ref:`laravel-fundamentals-insert-documents` section 
+   of the Write Operations guide.
 
-   .. output::
-      :language: console
-      :visible: false
+Example
+-------
 
-      Insert operation success: yes
+Select from the following :guilabel:`Eloquent` and :guilabel:`Query
+Builder` tabs to view usage examples for the same operation that use
+each corresponding query syntax:
+
+.. tabs::
+
+   .. tab:: Eloquent
+      :tabid: eloquent-model-count
+
+      This example performs the following actions:
+      
+      - Uses the ``Movie`` Eloquent model to represent the ``movies``
+        collection in the ``sample_mflix`` database 
+      - Inserts documents into the ``movies`` collection
+      - Prints whether the insert operation succeeds
+      
+      The example calls the ``insert()`` method to insert documents that model
```

Comment:
"model" here confused me here for a minute, I think because I was thinking "a model object" based on the previous bullets
```suggestion
      The example calls the ``insert()`` method to insert documents that represent
```